app/assets/nep5/raw-svg-export/index.js

```diff
@@ -0,0 +1,97 @@
+import NEO from '../NEO.svg'
```

Comment:
If you look in my changes to the webpack configurations this file actually bypasses the `react-svg-loader` by design for use in Nep9QrGenerator.js